multiqc/modules/vcftools/tstv_by_qual.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse TsTv by quality output from vcftools TsTv-by-qual """
+
+import csv
+import logging
+from collections import defaultdict
+from multiqc.plots import linegraph
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class TsTvByQualMixin():
+
+    def parse_tstv_by_qual(self):
+        """ Create the HTML for the TsTv by quality linegraph plot. """
+
+        self.vcftools_tstv_by_qual = dict()
+        for f in self.find_log_files('vcftools/tstv_by_qual', filehandles=True):
+            d = {}
+            for line in f['f'].readlines()[1:]: # don't add the header line (first row)
+                key = float(line.split()[0]) # taking the first column (QUAL_THRESHOLD) as key
+                val = float(line.split()[6]) # taking Ts/Tv_GT_QUAL_THRESHOLD as value
+                d[key] = val
+            self.vcftools_tstv_by_qual[f['s_name']] = d
+
+        # Filter out ignored sample names
+        self.vcftools_tstv_by_qual = self.ignore_samples(self.vcftools_tstv_by_qual)
+
+        if len(self.vcftools_tstv_by_qual) == 0:
+            return 0
+
+        pconfig = {
+            'id': 'vcftools_tstv_by_qual',
+            'title': 'TsTv by Qual',
+            'ylab': 'TsTv Ratio',
+            'xlab': 'SNP Quality Threshold'
```

Comment:
Suggested additional config options:

```python
'xmin': 0,
'ymin': 0,
'smooth_points': 400,
'smooth_points_sumcounts': False
```

This prevents the plot from autoscaling above zero when the range of differences is very small (it can be difficult to interpret otherwise, as you may have massive variance between `1.98` and `2.02` which looks important but isn't).

`smooth_points` is the most important as it limits how much data can be put into the report. This prevents huge filesizes and crashing the browser.